### ISSUE TITLE
[MU4 Issue] `Save a copy...` replaces active score with copy

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#10288]
**Describe the bug**
`File` > `Save a copy...` currently replaces the original working score with the newly saved (copy) score.

**To Reproduce**
1. Save a new score
2. Click `File` > `Save a copy...`, and give your score a different filename
3. Click `Save`
4. The score with the new filename is now active in the project window

**Expected behavior**
Per MS3.6, the new file created by `Save a copy...` should actually only be saved in the user's designated file directory, leaving the original file (i.e. not the new file) active in the project window.
In other words, `Save a copy...` should work as per MS3.6.

**Demo video**
https://www.dropbox.com/s/tjiuhsnod2jcm64/Save%20a%20copy%20demo.mov?dl=0

**Desktop (please complete the following information):**
 - MacOS 11.6 (M1)
 - Windows 11
